server/graphql/v1/mutations/tiers.js

```diff
@@ -6,6 +7,19 @@ export function editTiers(_, args, req) {
     throw new errors.Unauthorized('You need to be logged in to edit tiers');
   }
 
+  const tiers = map(args.tiers, tier => {
+    if (!tier.minimumAmount) {
```

Comment:
Why would `tier.minimumAmount` be falsey?

I see 2 cases:

- frontend is not updated yet and the behavior is right
- frontend is updated, is sending minimumAmount but it's empty because the admin didn't define a minimumAmount. In that case, I don't think we should re-process it.

Reply:
We should throw an error instead?

Reply:
Why would we throw an error? Is it mandatory?

Reply:
No it is not but we also can't have it as `null` since `amount` is mandatory.

Reply:
Once the feature is available on frontend, we should not mess with user submitted values.

If the user decide the minimumAmount is null, it should stay null.

Reply:
Alright, I can open another PR to remove that after they've both been deployed.

Reply:
The API code should be compatible with the old and new versions of frontend.